src/Cms/BindingProtocol.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Kirby\Cms;
+
+use Kirby\Http\Uri;
+use Kirby\Toolkit\Str;
+
+/**
+ * Handels parsing the protocol schema
+ * for the `Binding` class
+ *
+ * @package   Kirby Cms
+ * @author    Nico Hoffmann <nico@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ */
+class BindingProtocol extends Uri
+{
+	/**
+	 * supported schemes
+	 */
+	protected static array $schemes = [
+		'site',
+		'page',
+		'file',
+		'user',
+		'block',
+		'structure'
```

Comment:
Maybe stupid, but what about calling it struc://. Then we have about the same length as the rest and it's still easy to read.

Reply:
If, then I'd be for at least `struct://`